Mathlib/Topology/MetricSpace/HolderNorm.lean

```diff
@@ -61,7 +61,8 @@ lemma HolderWith.memHolder {C : ℝ≥0} (hf : HolderWith C r f) : MemHolder r f
   refine ⟨fun h => ?_,
     fun hf => let ⟨C, hC⟩ := hf; iInf_lt_top.2 ⟨C, iInf_lt_top.2 ⟨hC, coe_lt_top⟩⟩⟩
   simp_rw [eHolderNorm, iInf_lt_top] at h
-  exact let ⟨C, hC, _⟩ := h; ⟨C, hC⟩
+  let ⟨C, hC, _⟩ := h
```

Comment:
Can you use `obtain`, or do you really need a `let`?

Reply:
Why `obtain` when `let` is shorter?

Reply:
I have no preference either way.

Reply:
`let` is fine here.